sre.md

```diff
@@ -1,14 +1,15 @@
 # Specifications for Site Reliability Engineering with Keptn
 
 To support site reliability engineering with Keptn and to enable the self-healing use case, Keptn relies on two configurations called as:
-* [Service Level Objectives (SLO)](#service-level-objectives-(slo))
+* [Service Level Objectives (SLO)](#service-level-objectives-slo-configuration))

```

Comment:
there seems to be an extra closing `)` at the end - not caused by you PR, but can you still fix it please?

Reply:
It is gone.